lib/screens/wallet/components/tokens_cards/interactor/mappers/token_balances_state_mapper.dart

```diff
@@ -49,7 +49,7 @@ class TokenBalancesStateMapper {
     final repo = ColorPaletteRepository();
     for (final TokenBalanceViewModel viewModel in available) {
       if (viewModel.token != seedsToken) {
-        viewModel.dominantColor = await repo.getImagePaletteCached(viewModel.token.backgroundImage);
+        viewModel.dominantColor = await repo.getImagePalette(viewModel.token.backgroundImage);
```

Comment:
whats cached vs not cached?

Reply:
fixed at [c024231](https://github.com/JoinSEEDS/seeds_light_wallet/pull/1880/commits/c0242316552c0d26a5046308ed4707f1f0ef3f2e)